rules/integrations/google_workspace/persistence_google_workspace_2sv_policy_disabled.toml

```
@@ -27,7 +27,48 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "Google Workspace 2SV Policy Disabled"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Google Workspace 2SV Policy Disabled
+
+Google Workspace administrators manage password policies to enforce password requirements for an organization's compliance needs. Administrators have the capability to set restrictions on password length, reset frequencies, reuse capability, expiration, and much more. Google Workspace also allows multi-factor authentication (MFA) and 2-step verification (2SV) for authentication. 2SV allows users to verify their identity using security keys, Google prompt, authentication codes, text messages, and more.
+
+2SV adds an extra authentication layer for Google Workspace users to verify their identity. If 2SV or MFA aren't implemented, users only authenticate with their user name and password credentials. This authentication method has often been compromised and can be susceptible to credential access techniques when weak password policies are used.
+
+This rule detects when a 2SV policy is disabled in Google Workspace.
+
+#### Possible investigation steps
+
+- Identify the associated user account(s) by reviewing `user.name` or `source.user.email` in the alert.
+- Identify what password setting was created or adjusted by reviewing `google_workspace.admin.setting.name`.
+- Review if a password setting was enabled or disabled by reviewing `google_workspace.admin.new_value` and `google_workspace.admin.old_value`.
+- After identifying the involved user account, verify administrative privileges are scoped properly to change.
```

```suggestion
- After identifying the involved user account, verify administrative privileges are scoped properly.
```
We should either clarify what they are scoped properly to change, or just remove "to change".